R/set_epivars.R

```diff
@@ -0,0 +1,31 @@
+#' Define which columns are epivars in a linelist
+#'
+#' These functions can be used to change the `epivars` registred in a `linelist`
+#' object. `set_epivars` replaces previous `epivars` with new
+#' values. `add_epivars` adds new `epivars` to the existing ones.
+#'
+#' @param x a `linelist` object
+#'
+#' @param ... a series of column names named after epivars
+#' 
+#' @export
+
+set_epivars <- function(x, ...) {
+  dots <- valid_dots(list(...))
+  attr(x, "epivars") <- dots
+  x
```

Comment:
What's missing here is ordering the epivars so that they match the order of the data frame.